lib/ansible/modules/network/aci/aci_anp.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+
+module: aci_anp
+short_description: Manage top level application network profile objects
+description:
+    -  Manage top level application network profile object, i.e. this does
+      not manage EPGs.
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieesrs)
+version_added: '2.4'
+requirements:
+    - ACI Fabric 1.0(3f)+
+notes: Tenant must exist prior to using this module
```

Comment:
The `notes` is a list of items, and each note must end with a trailing dot.
Please also make this an full English sentence:
```yaml
notes:
- The tenant used must exist before using this module in your playbook. The M(aci_tenant) module can be used for this.
```